run.nextflow.sh

```diff
@@ -36,11 +37,13 @@ nextflow -C "${DIR}/nextflow-lib/nextflow.config" run "${DIR}/${pipeline}/${pipe
 ########################################################################################
 # Update submission receipt and submission metadata [as well as all the analyses archived]
 ########################################################################################
-"${DIR}/update.receipt.sh" "${batch_index}" "${snapshot_date}" "${pipeline}" "${profile}" "${root_dir}" "${dataset_name}" "${project_id}"
-"${DIR}/set.archived.sh" "${dataset_name}" "${project_id}"
+#"${DIR}/update.receipt.sh" "${batch_index}" "${snapshot_date}" "${pipeline}" "${profile}" "${root_dir}" "${dataset_name}" "${project_id}"
+#"${DIR}/set.archived.sh" "${dataset_name}" "${project_id}"
 
-rm -R "${pipeline_dir}/workDir" &
-rm -R "${pipeline_dir}/storeDir" &
-rm -R "${pipeline_dir}/publishDir" &
-wait
-rm -R "${pipeline_dir}"
+if [ "$profile" != "gls" ]; then
```

Comment:
I think that you would want to add one more condition "$profile" != "aws" or something to that sort. I have not seen Yanisa's impl. I do not know what she calls her profile. Please check with her.

Reply:
I think she can update it according to her implementation once she raises her PR? She would need to rebase with these changes anyway.

Reply:
OK. makes sense.